src/post.js

```diff
@@ -200,6 +200,8 @@ function assert_ok(fn, action) {
   } catch (e) {
     err = e;
   }
-  if (err !== 0)
+  // Allow HighsStatus::kWarning (0) and HighsStatus::kWarning (1) but 
+  // disallow other values, such as e.g. HighsStatus::kError (-1).
+  if (!(err in [0, 1]))
```

Comment:
```suggestion
  if (err !== 0 || err !== 1)
```

Reply:
@fuglede `in` doesn't do what you think on an array.

Reply:
I guess for this particular array it more or less does, but yeah, makes more sense to be explicit.